benchcab/benchcab.py

```diff
@@ -105,10 +105,10 @@ def _get_config(self, config_path: str) -> dict:
     def _get_models(self, config: dict) -> list[Model]:
         if not self._models:
             for id, sub_config in enumerate(config["realisations"]):
-                name = sub_config.get("name")
                 repo = create_repo(
                     spec=sub_config.pop("repo"),
-                    path=internal.SRC_DIR / name if name else internal.SRC_DIR,
+                    path=internal.SRC_DIR
+                    / (Path() if sub_config["name"] is None else sub_config["name"]),
```

Comment:
```suggestion
                    / (sub_config["name"] if sub_config["name"] else Path()),
```
I prefer it this way and it matches other places in the code that do the same.